crt_portal/cts_forms/templates/forms/snippets/preview_button.html

```diff
@@ -0,0 +1,3 @@
+<button name="wizard_goto_step" type="submit" label="edit {{ step_name }} step" value="{{ step_value }}" class="usa-button usa-button--outline margin-top-2" style="float:right; margin-top:2rem; margin-bottom:2rem;">
```

Comment:
I only see this and one other instance where we're using inline styles, not sure if there's project expectations around this but...should these styles be applied with a class?

We don't need to block merging for this but wanted to ask as I ramp up here 😄